Sources/DangerSwiftPeripherySampleApp/Dummy1.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  File.swift
+//  
+//
+//  Created by 多鹿豊 on 2022/04/05.
+//
+import Foundation
+
+struct UnusedStruct1 {
```

Comment:
<!--
  0 failure: 
  1 warning:  Struct 'UnusedStr...
  
  
  DangerID: danger-id-Danger;
  File: Sources/DangerSwiftPeripherySampleApp/Dummy1.swift;
  Line: 9;
-->

- :warning: Struct 'UnusedStruct1' is unused